bootique/src/main/java/io/bootique/help/config/HelpConfigCommand.java

```diff
@@ -48,7 +53,7 @@ public HelpConfigCommand(BootLogger bootLogger, Provider<ConfigHelpGenerator> he
 
     @Override
     public CommandOutcome run(Cli cli) {
-        List<String> arguments = cli.standaloneArguments();
+        List<String> arguments = cli.optionStrings(HELP_CONFIG_OPTION);
 
         Predicate<MetadataNode> predicate = (arguments.size() == 0)
                 ? o -> true
```

Comment:
Currently the CLI args are successfully being pulled in `run`. I've been trying to understand how this would all work when some modules other than `BQCoreModule` exist, so I turned to tests for easy setup.

I wanted to create some failing tests as a starting point, but I can't get the configs to not all print out. Even with a predicate of a lambda that always returns `false`, they still print out.

I think I currently have some muddying of testing framework understanding and bootique understanding, though, so ideally I'd just create these modules in my bootique run and devote some time to understanding the testing framework separately.